lib/checks/aria/aria-hidden-body-evaluate.js

```diff
@@ -1,5 +1,8 @@
-function ariaHiddenBodyEvaluate(node) {
-	return node.getAttribute('aria-hidden') !== 'true';
+function ariaHiddenBodyEvaluate(node, options, virtualNode) {
+	return (
+		virtualNode.hasAttr('aria-hidden') !== true ||
+		virtualNode.attr('aria-hidden') === 'false'
+	);
```

Comment:
Interesting change. Did this solve a problem you ran into? Was `virtualNode.attr('aria-hidden') !== 'true'` not working?

Reply:
My bad, fixed it. I was actually exploring the virtual-node js file and tried multiple functions of the same.